assets/scss/_settings.scss

```diff
@@ -0,0 +1,59 @@
+/*
+* Variable names are inspired by tailwind.css
+* e.g. ff = fontfamily; fs = fontsize; fs-xs = fontsize-xs
+* --------------------------------------------
+ */
+
+/*
+* Sizes (s)
+* Sizing has 4 properties: xs, s, l, xl; if no no size is applied it would be a m and it is the default value.
+ */
+$s-xxs: 0.25rem;
+$s-xs:  0.5rem;
+$s-s:   0.8rem;
+$s:     1rem;
+$s-m:   1.2rem;
+$s-l:   1.5rem;
+$s-xl:  2rem;
+
+$s-page-width: 210mm;
+$s-page-height: 297mm;
+
+/*
+* Fonts
+ */
+//font family (ff)
+$ff: 'Roboto', sans-serif;
+
+//font size (fs)
+$fs-xs: $s-xs;
+$fs-s:  $s-s;
+$fs:    $s;
+$fs-m:  $s-m;
+$fs-l:  $s-l;
+$fs-xl: $s-xl * 2;
+
+/*
+* Colors (c)
+ */
+$c-primary:        #2E2E3E;
+$c-font-headline:    $c-primary;
+$c-font-text:        rgba($c-primary, 0.8);
+$c-primary-shadow: rgba(18,18,24, 0.3);
+$c-secondary:      #E2EBF0;
+$c-accent:         rgba($c-primary, 0.6);
+$c-white:          #ffffff;
+$c-grey:           #cccccc;
+$c-black:          #000000;
+
+/*
+* Breakpoints
+ */
+$desktop: $s * 62;
```

Comment:
do you actually make use of adjusting $s somewhere? Otherwise I don't see a reason to confuse the readers. Write 62rem etc instead if you don't make use of changing $s somewhere. Otherwise ignore this comment.